rllib/models/specs/specs_dict.py

```diff
@@ -149,8 +149,8 @@ def __repr__(self) -> str:
 def check_specs(
     input_spec: str = "",
     output_spec: str = "",
-    filter: bool = True,
-    cache: bool = False,
+    filter: bool = False,
+    cache: bool = True,
```

Comment:
below in the docstring, there are a bunch of typos:
* 2 empty lines (should just be one)
* "return and a single object" ?
* -> change to "that has an arg named `input_data` in its args ..."
* -> "mappinga" ?

Reply:
Can we complete the dosctring with "Args:"?

Reply:
> Can we complete the dosctring with "Args:"?

What do you mean? it includes "Args:"